### PR TITLE
doc: release notes 3.6: ESP32 changes

### PR DESCRIPTION
**Pull request by sylvioalves**

### CODE REVIEW

**Review by MaureenHelm — DISMISSED** [review]

**Review by bjarki-andreasen — DISMISSED** [review]

**Review by henrikbrixandersen — DISMISSED** [review]

File: doc/releases/release-notes-3.6.rst [hunk 1]
```diff
@@ -130,6 +132,8 @@ Boards & SoC Support
 
 * Made these changes for Xtensa boards:
 
+  * Added ESP32S3 based M5Stack AtomS3 board: ``m5stack_atoms3``
```
Comment (kartben) [inline doc/releases/release-notes-3.6.rst:135]: can you please check, this doesn't sound right at all

```
git diff --name-only --diff-filter=A v3.5.0..HEAD -- boards/ | grep \.rst -i

...
boards/xtensa/esp32s2_lolin_mini/doc/index.rst
boards/xtensa/heltec_wireless_stick_lite_v3/doc/index.rst
boards/xtensa/kincony_kc868_a32/doc/index.rst
boards/xtensa/m5stack_atoms3/doc/index.rst
boards/xtensa/m5stack_atoms3_lite/doc/index.rst
boards/xtensa/m5stack_stamps3/doc/index.rst
```

File: samples/boards/esp32/flash_memory_mapped/README.rst [hunk 2]
```diff
@@ -1,4 +1,8 @@
-.. _flash_memory_mapped:
```
Comment (kartben) [inline samples/boards/esp32/flash_memory_mapped/README.rst:1]: great call! :) 
But then please remove lines 7-9 as the title is auto-added by the code-sample thingy.

**Review by kartben — CHANGES_REQUESTED** [review]

File: doc/releases/release-notes-3.6.rst (hunk 1 above)
Comment (sylvioalves) [reply doc/releases/release-notes-3.6.rst]: Aha, sorry, thanks!

Comment (kartben) [reply doc/releases/release-notes-3.6.rst]: wait, there's now ... zero board in your PR? :)

**Review by henrikbrixandersen — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Pull request merged**